storefront/app/views/themes/default/spree/shared/icons/_menu.html.erb

```diff
@@ -1,3 +1,5 @@
+<% color ||= 'black' %>
```

Comment:
```suggestion
<% color ||= theme_setting('text_color') %>
```

Reply:
This way it will always use the color set for a theme in page builder.

Also, thank you for the contribution!